README.rst

```diff
@@ -23,20 +23,16 @@
 .. _Gitter: https://gitter.im/matplotlib/matplotlib
 
 .. |NUMFocus| image:: https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
-.. _NUMFocus: http://www.numfocus.org
+.. _NUMFocus: https://www.numfocus.org
 
-
-.. |GitTutorial| image:: https://img.shields.io/badge/PR-Welcome-%23FF8300.svg?
-.. _GitTutorial: https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project
```

Comment:
I don't think you want to remove this, because it's still there on line 1.

Reply:
🤦‍♀️ clearly I shouldn't do backports